<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ceusAnalysis</class>
 <widget class="QWidget" name="ceusAnalysis">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1175</width>
    <height>749</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Select Region of Interest</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
	background: rgb(42, 42, 42);
}</string>
  </property>
  <widget class="QLabel" name="curSliceLabel">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>560</y>
     <width>361</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	font-size: 20px;
	color: rgb(255, 255, 255);
	background-color: rgba(255, 255, 255, 0);
}</string>
   </property>
   <property name="text">
    <string>Current Frame:</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QSlider" name="curSliceSlider">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>620</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="curSliceOfLabel">
   <property name="geometry">
    <rect>
     <x>455</x>
     <y>680</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	font-size: 17px;
	color: rgb(255, 255, 255);
	background-color: rgba(255, 255, 255, 0);
}</string>
   </property>
   <property name="text">
    <string>of</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::NoTextInteraction</set>
   </property>
  </widget>
  <widget class="QWidget" name="sidebar" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>341</width>
     <height>751</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget {
	background-color: rgb(28, 0, 101);
}</string>
   </property>
   <widget class="QFrame" name="imageSelectionSidebar">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>341</width>
      <height>121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame {
	background-color: rgb(99, 0, 174);
	border: 1px solid black;
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="imageSelectionLabelSidebar">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>0</y>
       <width>191</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
	font-size: 21px;
	color: rgb(255, 255, 255);
	background-color: rgba(255, 255, 255, 0);
	border: 0px;
	font-weight: bold;
}</string>
     </property>
     <property name="text">
      <string>Image Selection:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="imageLabel">
     <property name="geometry">
      <rect>
       <x>-60</x>
       <y>50</y>
       <width>191</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
	font-size: 16px;
	color: rgb(255, 255, 255);
	background-color: rgba(255, 255, 255, 0);
	border: 0px;
	font-weight: bold;
}</string>
     </property>
     <property name="text">
      <string>Image:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="imagePathInput">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>50</y>
       <width>241</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
	font-size: 14px;
	color: rgb(255, 255, 255);
	background-color: rgba(255, 255, 255, 0);
	border: 0px;
}</string>
     </property>
     <property name="text">
      <string>Sample filename </string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="roiSidebar">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>120</y>
      <width>341</width>
      <height>121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame {
	background-color: rgb(99, 0, 174);
	border: 1px solid black;
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="roiSidebarLabel">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>341</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
	font-size: 21px;
	color: rgb(255, 255, 255);
	background-color: rgba(255, 255, 255, 0);
	border: 0px;
	font-weight: bold;
}</string>
     </property>
     <property name="text">
      <string>Region of Interest (ROI) Selection</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="rfAnalysisSidebar">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>480</y>
      <width>341</width>
      <height>121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame {
	background-color:  rgb(49, 0, 124);
	border: 1px solid black;
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="rfAnalysisLabel">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>341</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
	font-size: 21px;
	color: rgb(255, 255, 255);
	background-color: rgba(255, 255, 255, 0);
	border: 0px;
	font-weight: bold;
}</string>
     </property>
     <property name="text">
      <string>Export Results</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="ticAnalysisSidebar">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>360</y>
      <width>341</width>
      <height>121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame {
	background-color:  rgb(99, 0, 174);
	border: 1px solid black;
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="ticAnalysisLabel">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>341</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
	font-size: 21px;
	color: rgb(255, 255, 255);
	background-color: rgba(255, 255, 255, 0);
	border: 0px;
	font-weight: bold;
}</string>
     </property>
     <property name="text">
      <string>Contrast-Enhanced Ultrasound
(CEUS) Analysis</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="saveDataButton">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>630</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	color: white;
	font-size: 12px;
	background: rgb(90, 37, 255);
	border-radius: 15px;
}
QPushButton:checked {
	color:white; 
	font-size: 12px;
	background: rgb(45, 0, 110);
	border-radius: 15px;
}</string>
    </property>
    <property name="text">
     <string>Save Data</string>
    </property>
   </widget>
   <widget class="QPushButton" name="exportDataButton">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>690</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	color: white;
	font-size: 12px;
	background: rgb(90, 37, 255);
	border-radius: 15px;
}
QPushButton:checked {
	color:white; 
	font-size: 12px;
	background: rgb(45, 0, 110);
	border-radius: 15px;
}</string>
    </property>
    <property name="text">
     <string>Export Data</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="analysisParamsSidebar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>240</y>
     <width>341</width>
     <height>121</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame {
	background-color: rgb(99, 0, 174);
	border: 1px solid black;
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="analysisParamsLabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>341</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
	font-size: 21px;
	color: rgb(255, 255, 255);
	background-color: rgba(255, 255, 255, 0);
	border: 0px;
	font-weight:bold;
}</string>
    </property>
    <property name="text">
     <string>TIC Modification</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="backButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>690</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	color: white;
	font-size: 16px;
	background: rgb(90, 37, 255);
	border-radius: 15px;
}</string>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="curSliceSpinBox">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>685</y>
     <width>48</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QSpinBox {
	background: white;
	color: black;
}</string>
   </property>
  </widget>
  <widget class="QLabel" name="constructRoiLabel">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>-20</y>
     <width>431</width>
     <height>131</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	font-size: 29px;
	color: rgb(255, 255, 255);
	background-color: rgba(255, 255, 255, 0);
}</string>
   </property>
   <property name="text">
    <string>CEUS Analysis Results:</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="ceCoverLabel">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>140</y>
     <width>381</width>
     <height>351</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	background-color: transparent;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="peVal">
   <property name="geometry">
    <rect>
     <x>1100</x>
     <y>560</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	color: white;
	background-color: rgba(0,0,0,0);
}</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="tpLabel">
   <property name="geometry">
    <rect>
     <x>1040</x>
     <y>600</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	color: white;
	background-color: rgba(0,0,0,0);
}</string>
   </property>
   <property name="text">
    <string>TP</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="voiVolumeVal">
   <property name="geometry">
    <rect>
     <x>1100</x>
     <y>650</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	color: white;
	background-color: rgba(0,0,0,0);
}</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="mttLabel">
   <property name="geometry">
    <rect>
     <x>1040</x>
     <y>580</y>
     <width>81</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	color: white;
	background-color: rgba(0,0,0,0);
}</string>
   </property>
   <property name="text">
    <string>MTT</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="tpVal">
   <property name="geometry">
    <rect>
     <x>1100</x>
     <y>600</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	color: white;
	background-color: rgba(0,0,0,0);
}</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="voiVolumeLabel">
   <property name="geometry">
    <rect>
     <x>1040</x>
     <y>640</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	color: white;
	background-color: rgba(0,0,0,0);
}</string>
   </property>
   <property name="text">
    <string>ROI Area
(mm^2)</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QFrame" name="ticDisplay">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>560</y>
     <width>401</width>
     <height>171</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QLabel" name="aucVal">
   <property name="geometry">
    <rect>
     <x>1100</x>
     <y>540</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	color: white;
	background-color: rgba(0,0,0,0);
}</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="peLabel">
   <property name="geometry">
    <rect>
     <x>1040</x>
     <y>560</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	color: white;
	background-color: rgba(0,0,0,0);
}</string>
   </property>
   <property name="text">
    <string>PE</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="resultsLabel">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>510</y>
     <width>361</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	font-size: 20px;
	color: rgb(255, 255, 255);
	background-color: rgba(255, 255, 255, 0);
}</string>
   </property>
   <property name="text">
    <string>Results:</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="mttVal">
   <property name="geometry">
    <rect>
     <x>1100</x>
     <y>580</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	color: white;
	background-color: rgba(0,0,0,0);
}</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="aucLabel">
   <property name="geometry">
    <rect>
     <x>1040</x>
     <y>540</y>
     <width>81</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	color: white;
	background-color: rgba(0,0,0,0);
}</string>
   </property>
   <property name="text">
    <string>AUC</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="curSliceTotal">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>680</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	font-size: 17px;
	color: rgb(255, 255, 255);
	background-color: rgba(255, 255, 255, 0);
}</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::NoTextInteraction</set>
   </property>
  </widget>
  <widget class="QLabel" name="imPlane">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>80</y>
     <width>691</width>
     <height>421</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="t0Label">
   <property name="geometry">
    <rect>
     <x>1040</x>
     <y>620</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	color: white;
	background-color: rgba(0,0,0,0);
}</string>
   </property>
   <property name="text">
    <string>T0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="t0Val">
   <property name="geometry">
    <rect>
     <x>1100</x>
     <y>620</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	color: white;
	background-color: rgba(0,0,0,0);
}</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="maskCoverLabel">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>80</y>
     <width>691</width>
     <height>421</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
background-color: rgba(255, 255, 255, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
